VeroliBGV/BGV009.xml

```diff
@@ -575,10 +575,29 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            <locus target="#75vb"/>
                            <hi rend="rubric">አመ</hi>፡ አድባራት፡ ሮጸ፡ እምቅዋሞን፡ ወአመ፡ እምድሮን፡ አንቀ
                         </explicit>
-                        <note>The text is written in another hand (cf. <ref target="#h2"/>) on an originally blank leaf. It continues on <ref target="#p2"/> (<locus from="76ra" to="77ra"/>).</note>
+                        <note>
+                           The text is written in another hand (cf. <ref target="#h2"/>) on an originally blank leaf. It continues on <ref target="#p3"/> (<locus from="76ra" to="77ra"/>).
```

Comment:
We should refer to this fascinating scribal practice across diefferent codicological units in our introduction.